### PR TITLE
[RFR] default idFields to id

### PR DESCRIPTION
**Pull request by ThieryMichel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ Creates a query to select one entity.
 #### Configuration
 
 - table: the table name
-- idFields: list of key fields used to select the entity
+- idFields: list of key fields used to select the entity, default to `id`
```
Comment (djhi) [inline README.md]: s/default to `id`/(default: `id`)/

**Comment by djhi** [comment]
Might be a good time to rename this parameter to `filters` (see #8) ?

**Comment by ThieryMichel** [comment]
filters would not be a good name IMO (see #8)

**Comment by ThieryMichel** [comment]
Besides there is lot of renaming to be done, I'd rather do it in a dedicated PR.

**Pull request merged**